packages/transformer/src/IModelImporter.ts

```diff
@@ -229,6 +259,25 @@ export class IModelImporter {
 
   /** Import the specified ElementProps (either as an insert or an update) into the target iModel. */
   public importElement(elementProps: ElementProps): Id64String {
+    const tryUpdateElement = (elemProps: ElementProps): void => {
```

Comment:
[NitPick] Maybe its is just preference, but methods are becoming very long.
I think this part (as many others) would benefit if code was split into smaller separate functions e.g. in this case, why not move this to private method, instead of having it as local variable.

Reply:
Added